ibu-imager/cmd/create.go

```diff
@@ -88,6 +89,14 @@ func create() error {
 	op := ops.NewOps(log, hostCommandsExecutor)
 	rpmOstreeClient := ostree.NewClient("ibu-imager", hostCommandsExecutor)
 
+	defer func() {
+		if err = seedrestoration.NewSeedRestoration(log, op, common.BackupDir, containerRegistry,
```

Comment:
Lets add command for seed-restore too, can be done in next pr

Reply:
Agree let's merge this first